src/widgets/Dialogs.js

```diff
@@ -180,7 +180,7 @@ define(function (require, exports, module) {
             }
         } else if (brackets.platform === "mac") {
             // CMD+D Don't Save
```

Comment:
From what I can tell (not that I have a Mac...), it's Cmd+Delete:
![image](https://cloud.githubusercontent.com/assets/2641501/5992320/ba5a6c92-aa24-11e4-8bd8-00da4d6168f9.png)